src/twin/system_info.rs

```diff
@@ -21,15 +74,24 @@ lazy_static! {
 }
 
 #[derive(Default, Serialize)]
-pub struct SystemInfo {
-    #[serde(skip_serializing)]
-    tx_reported_properties: Option<mpsc::Sender<serde_json::Value>>,
+pub struct Info {
     os: serde_json::Value,
     azure_sdk_version: String,
     omnect_device_service_version: String,
     boot_time: Option<String>,
 }
 
+#[derive(Default)]
+pub struct SystemInfo {
```

Comment:
rename to `System`

Reply:
don't like to change the name because there is also a file called system.rs under the src folder

Reply:
well but that's not unusual. I propose to move the remaining code in system.rs to main.rs

Reply:
done